src/Tools/dotnet-monitor/dotnet-monitor.csproj

```diff
@@ -13,17 +13,9 @@
     <!-- This forces the creation of a checksum file and uploads it to blob storage
          using this name as part of the blob relative path. -->
     <BlobGroupPrefix>monitor</BlobGroupPrefix>
-    <!-- File locking warnings (that are typically resolved with automatic retries) are failing builds
-         quite frequently with messages such as:
-         
-         error MSB3026: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not copy "...\dotnet-monitor.dll" to
-         "...\dotnet-monitor.dll". Beginning retry 1 in 1000ms. The process cannot access the file
-         '...\dotnet-monitor.dll' because it is being used by another process.
-         
-         work around this problem by telling the compiler to not warn about the issue. A file lock that
-         actually prevents the build from completing correctly should still fail the compilation with
-         an MSB3027 error. -->
-    <NoWarn>$(NoWarn);MSB3026</NoWarn>
+    <!-- Temporarily pin netcoreapp3.1 runtime framework version until .NET 6 GA.
+         This will allow the tool to be built without the unreleased minimum runtime framework versions. -->
+    <RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp3.1'">3.1.0</RuntimeFrameworkVersion>
```

Comment:
I'm a little confused by this. If you target 3.1 wouldn't the default minimum be 3.1?

Reply:
Discussed offline